Client/src/Components/Graph/Graph.js

```diff
@@ -51,51 +32,50 @@ datalist.push(dataset3)
 datalist.push(dataset4)
 
 class Graph extends Component {
+
     componentDidMount() {
         this.graphCreation(datalist)
     }
     graphCreation(datalist) {
         //Calls the function to create the axis
-        startLinear();
+        //startLinear();
```

Comment:
You can just remove the function if it is not used. A new function will be added later for the ln to log etc.